.travis.yml

```diff
@@ -1,18 +1,11 @@
 language: generic
 
 matrix:
-  include:
-    - os: linux
-      python: 3.6
-      env:
-        - PYTHON_VERSION=3.6
-        - CTAPIPE_VERSION=v0.7.0
-
     - os: linux
       python: 3.7
       env:
         - PYTHON_VERSION=3.7
-        - CTAPIPE_VERSION=v0.7.0
+        - CTAPIPE_VERSION=master
```

Comment:
why not leaving the test agains the latest release plus that of the master?

Reply:
Because the first change we will make to use the master will break compatiblity with 0.7, so why keep testing against that?

Or do you want to keep compatibility to both? That will be a lot of work and pretty ugly.

Reply:
whenever ctapipe v0.8.0 is released, we can update the version of the latest release, but if we only leave the `master` in travis, tests will not pass whenever they do not become compatible

Reply:
E.g. ctapipe.io.containers → ctapipe.containers

Reply:
As soon as 0.8 is released, 0.8 and master should be tested

Reply:
ok, I see your point, I would still like to keep a reminder that whenever this is merged into ctapipe_io_lst `master`, we should add the ctapipe `v0.8.0` test -> maybe comment it in?

Reply:
In fact, this should be done **before** this is merged, to show that this is really compatible with 0.8.

Reply:
so what about if instead of deleting, we comment those lines?

Reply:
Ok, I simply added the test for ctapipe 0.8. It will fail until ctapipe 0.8 is released